src/main/java/mate/academy/Main.java

```diff
@@ -1,16 +1,162 @@
 package mate.academy;
 
+import java.time.LocalDate;
+import mate.academy.lib.Injector;
+import mate.academy.model.CinemaHall;
 import mate.academy.model.Movie;
+import mate.academy.model.MovieSession;
+import mate.academy.service.CinemaHallService;
 import mate.academy.service.MovieService;
+import mate.academy.service.MovieSessionService;
 
 public class Main {
+    private static final Injector injector = Injector.getInstance("mate.academy");
+
     public static void main(String[] args) {
-        MovieService movieService = null;
 
-        Movie fastAndFurious = new Movie("Fast and Furious");
-        fastAndFurious.setDescription("An action film about street racing, heists, and spies.");
-        movieService.add(fastAndFurious);
-        System.out.println(movieService.get(fastAndFurious.getId()));
-        movieService.getAll().forEach(System.out::println);
+        // 0. INJECTING
+        final var cinemaHallService
+                = (CinemaHallService) injector.getInstance(CinemaHallService.class);
+        final var movieService
+                = (MovieService) injector.getInstance(MovieService.class);
+        final var movieSessionService
+                = (MovieSessionService) injector.getInstance(MovieSessionService.class);
+
+        // 1. CREATING
+        // Movies
+        Movie movie1 = new Movie();
+        movie1.setTitle("The Godfather");
+        movie1.setDescription("The Godfather");
+
+        Movie movie2 = new Movie();
+        movie2.setTitle("The Godfather");
+        movie2.setDescription("The Godfather");
+
+        Movie movie3 = new Movie();
+        movie3.setTitle("The Godfather");
+        movie3.setDescription("The Godfather");
+
+        // CinemaHalls
+        CinemaHall cinemaHall1 = new CinemaHall();
+        cinemaHall1.setCapacity(100);
+        cinemaHall1.setDescription("Hall 1");
+
+        CinemaHall cinemaHall2 = new CinemaHall();
+        cinemaHall2.setCapacity(100);
+        cinemaHall2.setDescription("Hall 1");
+
+        CinemaHall cinemaHall3 = new CinemaHall();
+        cinemaHall3.setCapacity(100);
+        cinemaHall3.setDescription("Hall 1");
+
+        //MovieSessions
+        MovieSession movieSession1 = new MovieSession();
+        movieSession1.setMovie(movie1);
+        movieSession1.setCinemaHall(cinemaHall1);
+        movieSession1.setShowTime(2024, 12,1,10, 15);
+
+        MovieSession movieSession2 = new MovieSession();
+        movieSession2.setMovie(movie2);
+        movieSession2.setCinemaHall(cinemaHall2);
+        movieSession2.setShowTime(2024, 12,1,10, 15);
+
+        MovieSession movieSession3 = new MovieSession();
+        movieSession3.setMovie(movie3);
+        movieSession3.setCinemaHall(cinemaHall3);
+        movieSession3.setShowTime(2024, 12,1,10, 15);
```

Comment:
The `setShowTime` method is likely expecting a `LocalDateTime` object rather than separate integers. Consider using `LocalDateTime.of(year, month, day, hour, minute)` to create a `LocalDateTime` object.